app/models/claim/advocate_hardship_claim.rb

```diff
@@ -23,8 +23,11 @@ class AdvocateHardshipClaim < BaseClaim
                    unless: proc { |c| c.disable_for_state_transition.eql?(:all) }
     validates_with ::Claim::AdvocateHardshipClaimSubModelValidator
 
+<<<<<<< HEAD
```

Comment:
rebase gone wrong here @kmahern

Reply:
👍 this has been removed

Reply:
but there should only be one commit then